paddle/fluid/operators/fake_quantize_op.cc

```diff
@@ -490,20 +499,68 @@ And it will not quantize the input tensor.
   }
 };
 
+class FakeQuantDequantGradOp : public framework::OperatorWithKernel {
+ public:
+  using framework::OperatorWithKernel::OperatorWithKernel;
+
+  void InferShape(framework::InferShapeContext* ctx) const override {
+    auto out_grad_name = framework::GradVarName("Out");
+    OP_INOUT_CHECK(ctx->HasInput(out_grad_name), "Input", out_grad_name,
+                   "FakeQuantDequantGradOp");
+
+    auto x_grad_name = framework::GradVarName("X");
+    PADDLE_ENFORCE_EQ(
```

Comment:
Why not use OP_INOUT_CHECK

Reply:
Can I fix it in the next PR? Because CI runs very slowly.

Reply:
I agree with it. Please also add us as reviewer in the next PR.

Reply:
Agree.